docs/contributing_guidelines.md

```diff
@@ -13,7 +13,7 @@ slug: /Contribution/contributing_guidelines
 * When submitting a PR with changes to user interface (e.g.: new screen, ...), please add screenshots to the PR description.
 * Before committing your changes, **apply** black auto-format - `black {name_of_the_file.py}` for specific file or `black .` to reformat all files
 * When you are finished with your work, please squash your commits otherwise we will squash them on your PR (this can help us maintain a clear commit history). 
-* When creating an issue to report a bug in the project, please follow our [bug_report.md](https://raw.githubusercontent.com/anitab-org/stem-diverse-tv/master/.github/ISSUE_TEMPLATE/bug_report.md) template.
+* When creating an issue to report a bug in the project, please follow our [bug_report.md](https://github.com/anitab-org/stem-diverse-tv/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) template.
```

Comment:
Please use the the templets given [here](https://github.com/anitab-org/.github)

Reply:
I didn't understand. You mean to edit the PR description and use the template?

Reply:
```suggestion
* When creating an issue to report a bug in the project, please follow our [bug_report.md](https://github.com/anitab-org/.github/blob/main/.github/ISSUE_TEMPLATE/bug_report.md) template.
```

Reply:
@lem0n4id replace the link with the suggested link.